Firmware/hummingbot_firmware/Hummingbot_firmware_FreeRTOS_2/source/Hummingbot_firmware_FreeRTOS_2.c

```diff
@@ -85,27 +111,86 @@ typedef struct{
   pin_t 		rf24_ce;
   char 			rf24_address[RF24_COMMON_ADDRESS_SIZE];
 }Hummingbot_firmware_FreeRTOS_2_S;
+//TODO: change name of struct??
+
+typedef struct{
+	float int1;
```

Comment:
@leekenjen2328 Preferably to avoid float, the FPU is slow, might get blocked by FTM pwm